### PR TITLE
Lambda hangs while converting some files

### PR DESCRIPTION
**Pull request by golenkovm**
This fix will mitigate issue #21 from plugin side

### CODE REVIEW

**Comment by golenkovm** [comment]
Conversation timeout is set to 1 hour by default.
Can be adjusted on librelambda setting page.
When timeout is reached status is set to conversion::STATUS_FAILED.

File: lang/en/fileconverter_librelambda.php [hunk 1]
```diff
@@ -50,6 +50,8 @@
 $string['settings:deletesuccess'] = 'Could delete object from the external object storage - It is not recommended for the user to have delete permissions. ';
 $string['settings:deleteerror'] = 'Could not delete object from the external object storage. ';
 $string['settings:permissioncheckpassed'] = 'Permissions check passed.';
+$string['settings:conversation_timeout'] = 'Conversation timeout';
```
Comment (dmitriim) [inline lang/en/fileconverter_librelambda.php]: Typo? Conversation-> Conversion

**Comment by dmitriim** [comment]
Created travis related issue https://github.com/catalyst/moodle-fileconverter_librelambda/issues/23

File: lang/en/fileconverter_librelambda.php (hunk 1 above)
Comment (golenkovm) [reply lang/en/fileconverter_librelambda.php]: Yeah. Fixed. Thanks

**Pull request merged**